lib/Util/AutoPagingIterator.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Stripe\Util;
+
+class AutoPagingIterator implements \Iterator
+{
+    private $page = null;
+    private $params = array();
+
```

Comment:
Nitpick, but maybe just clean up the whitespace around these parameters and alphabetized them?